dts/bindings/flash_controller/atmel,sam-flash-controller.yaml

```diff
@@ -10,3 +10,11 @@ include: flash-controller.yaml
 properties:
   clocks:
     required: true
+
+  "#flash-page-cells":
+    type: int
+    const: 2
```

Comment:
It will be necessary the documentation with examples. Remember that devicetree should be self explanatory and readers should not go to the datasheet to have a full understanding.

Reply:
Agreed